project-base/storefront/schema-compress.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+const fs = require('fs');
+const path = require('path');
+const introspection = require('@urql/introspection');
+
+const schemaFilePath = path.resolve(__dirname, './schema.graphql.json');
+const schemaFileContent = fs.readFileSync(schemaFilePath, 'utf8');
+const schema = JSON.parse(schemaFileContent);
+const result = introspection.minifyIntrospectionQuery(schema);
+const minifiedJson = JSON.stringify(result);
+
+fs.writeFileSync('./schema-compressed.graphql.json', minifiedJson, 'utf8');
```

Comment:
Can you please add deletion of the original uncompressed file? So during the development, the developer will not be forced to resolve conflicts in the unnecessary file.
Also the original uncompressed file should be ignored by git

Reply:
Done